Sources/MountebankSwift/Models/Imposter/Imposter.swift

```diff
@@ -3,13 +3,54 @@ import Foundation
 /// Imposter as documented on:
 /// [mbtest.org/docs/api/contracts?type=imposter](https://www.mbtest.org/docs/api/contracts?type=imposter)
 public struct Imposter: Codable, Equatable {
+
+    /// Mountebank does also support `tcp`, `smtp` and custom protocols
+    /// implemented in community plugins.
+    ///
+    /// Please submit a feature request issue on Github for support if you need other protocols
+    public enum ImposterNetworkProtocol: Codable, Equatable {
```

Comment:
If nested inside of `Imposter` this could just be 

```suggestion
    public enum NetworkProtocol: Codable, Equatable {
```

Right?

Reply:
Good point indeed possible when inside the imposter itself;